src/ConnectedMode.UnitTests/Migration/ConnectedModeMigrationTests.cs

```diff
@@ -55,7 +59,7 @@ public async Task Migrate_NoFilesToClean_DirectoryIsDeleted()
 
             var testSubject = CreateTestSubject(fileProvider.Object, fileCleaner.Object, fileSystem.Object);
 
-            await testSubject.MigrateAsync(null, CancellationToken.None);
+            await testSubject.MigrateAsync(AnyBoundProject, null, CancellationToken.None);
```

Comment:
The migration process doesn't do anything with the binding information yet so there are no tests for it actually using it.

It will use it when it starts calculating the legacy settings (separate ticket).